Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -207,11 +238,15 @@ void GeneralWidget::AddDescriptions()
       QT_TR_NOOP("Uses the main Dolphin window for rendering rather than "
                  "a separate render window.<br><br><dolphin_emphasis>If unsure, leave "
                  "this unchecked.</dolphin_emphasis>");
-  static const char TR_ASPECT_RATIO_DESCRIPTION[] = QT_TR_NOOP(
-      "Selects which aspect ratio to use when rendering.<br><br>Auto: Uses the native aspect "
-      "ratio<br>Force 16:9: Mimics an analog TV with a widescreen aspect ratio.<br>Force 4:3: "
-      "Mimics a standard 4:3 analog TV.<br>Stretch to Window: Stretches the picture to the "
-      "window size.<br><br><dolphin_emphasis>If unsure, select Auto.</dolphin_emphasis>");
+  static const char TR_ASPECT_RATIO_DESCRIPTION[] =
+      QT_TR_NOOP("Selects which aspect ratio to use when rendering.<br>"
+                 "Each game can have a slightly different default aspect ratio."
+                 "<br><br>Auto: Uses the native aspect ratio"
```

Comment:
Is there a reason why what I assume is the same concept is first referred to as "default aspect ratio" and then "native aspect ratio"?